UnitTests/homework-0209-3.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using NUnit.Framework;
+
+namespace UnitTests
+{
+    public class NowIs13HTest
+    {
+        [Test]
+        public void NowIs13HMethod()
+        {
+            var now = DateTime.Now;
```

Comment:
Jei turite kintamąjį `expectedTime`, tai kintamąjį `now` būtų logiška vadinti `nowTime`. Ir šiaip taip būtų logiška, nes naudojate `DateTime` tipą, tai jis ir grąžina tik laiką, be datos.

Reply:
Meluoju... :) Supratau dėl ko naudojate `now`, bet čia kažkaip sudėtingai tuomet šis testas atrodo.